repository-data/development/src/main/resources/hcm-content/content/documents/lks/home.yaml

```diff
@@ -4,17 +4,19 @@
   jcr:mixinTypes: ['hippo:named', 'hippo:versionInfo', 'mix:referenceable']
```

Comment:
I think `/hee:HeroSection` needs to be cleaned up from this yaml as this has been replaced by `Page title` & `Summary`

Reply:
done